### PR TITLE
OpensDistro Release 1.8.1

### PR DESCRIPTION
**Pull request by joshuali925**
*Issue #, if available:*
- #84 

*Description of changes:*
- OpenDistro Release 1.8.1.0

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by chloe-zh — APPROVED** [review]
LGTM, thanks!

File: release-notes/sql-jdbc.release-notes-1.8.1.0.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## 2020-06-15, Version 1.8.1.0 (Current)
+### Changes
```
Comment (chloe-zh) [inline release-notes/sql-jdbc.release-notes-1.8.1.0.md]: Could you change it to "Features" instead

Comment (joshuali925) [reply release-notes/sql-jdbc.release-notes-1.8.1.0.md]: Sure, changed

**Comment by joshuali925** [comment]
Skipping 1.8.1, release for 1.9 instead.

**Pull request closed without merging**